src/styles/index.scss

```diff
@@ -1,3 +1,4 @@
-body {
-  margin: 0;
-}
+@use 'sass:list';
+@import 'variables';
+@import 'page';
```

Comment:
Consider replacing `@import 'page';` with `@use 'page';` to follow the modern Sass practices.